core/src/main/resources/com/linecorp/armeria/public_suffixes.txt

```diff
@@ -4832,6 +4832,8 @@ loseyourip.com
 loten.no
 lotte
 lotto
+lovable.app
+lovableproject.com
```

Comment:
Should we revert this change? That will be automatically updated by a GitHub workflow.